components/contribution-flow/ContributionFlowSuccess.js

```diff
@@ -247,12 +247,21 @@ class ContributionFlowSuccess extends React.Component {
                   <P fontSize="20px" color="black.700" fontWeight={500} textAlign="center">
                     <FormattedMessage
                       id="NewContributionFlow.Success.NowSupporting"
-                      defaultMessage="You are now supporting {collective}."
-                      values={{ collective: order.toAccount.name }}
+                      defaultMessage="You are now supporting <link>{collective}</link>."
+                      values={{
+                        collective: order.toAccount.name,
+                        link: value => <Link href={{ pathname: order.toAccount.slug }}>{value}</Link>,
+                      }}
                     />
                   </P>
                 </Box>
-                <ContributorCardWithTier width={250} height={380} contribution={order} my={2} useLink={!isEmbed} />
+                {isEmbed ? (
+                  <ContributorCardWithTier width={250} height={380} contribution={order} my={2} useLink={false} />
+                ) : (
+                  <Link href={{ pathname: order.toAccount.slug }}>
```

Comment:
This changes the font styles in some parts of the card, the result is a bit surprising.

![Peek 2021-06-16 09-24](https://user-images.githubusercontent.com/1556356/122176414-d9cc9380-ce84-11eb-875e-a5cb7d31897c.gif)

To be tested, but I think this can be resolved by doing something like:

```suggestion
                  <StyledLink as={Link} color="black.800" href={{ pathname: order.toAccount.slug }}>
```

Reply:
Done👍🏻